src/Illuminate/Database/Seeder.php

```diff
@@ -28,9 +28,10 @@ abstract class Seeder
      *
      * @param  array|string  $class
      * @param  bool  $silent
+     * @param  mixed ...$params
      * @return $this
      */
-    public function call($class, $silent = false)
+    public function call($class, $silent = false, ...$params)
```

Comment:
This is a breaking change to the method signature unfortunately.

Reply:
Even if the parameter is optional, it's considered a breaking change?

Reply:
The method is part of the public API so if anyone's overriding it they'll see their code break: https://3v4l.org/hrke5

Reply:
I see, I thought php handled variadic parameters as optional parameters, doesn't seem like it does so I guess I could use `func_get_args`?

Reply:
I mage the change so it's not breaking anymore

Reply:
It's best that you send in a new one.

Reply:
Ok will do